README.md

```diff
@@ -38,6 +38,7 @@ For testing your patch locally follow the steps given below,
 For a good visualisation of the different data structures and algorithms, refer the following websites:
 - https://visualgo.net/
 - https://www.cs.usfca.edu/~galles/visualization/
+- [https://github.com/adavijit/BlogMan](https://github.com/adavijit/BlogMan)
```

Comment:
This is not the URL which was to be added. Please see the issue description of #264

Reply:
Oh sorry, I thought I had the URL copied. 
I have made the changes.